### PR TITLE
OPENEUROPA-3378: Fix date widget format and use inline entity form as entity reference widget

### PR DESCRIPTION
**Pull request by julien-**
Related to SD team asked changes on https://github.com/openeuropa/oe_theme/pull/507

### CODE REVIEW

File: modules/oe_content_project/config/install/core.entity_form_display.node.oe_project.default.yml [hunk 1]
```diff
@@ -115,7 +115,7 @@ content:
       size: 60
       placeholder: ''
```
Comment (ademarco) [inline modules/oe_content_project/config/install/core.entity_form_display.node.oe_project.default.yml:116]: These seem to be autocomplete widget settings, not inline entity form ones, did you re-export this config after changing it from UI or just replaced entity_reference_revisions_autocomplete with inline_entity_form_simple in here manually?

**Comment by ademarco** [comment]
@julien- also, Behat tests are not updated.

File: tests/features/project.feature [hunk 2]
```diff
@@ -40,10 +23,35 @@ Feature: Project content creation
...
     And I fill in "Teaser" with "Project teaser text" in the "Alternative titles and teaser" region
     And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
+
+    # Fill in Stakeholder fields by inline entity form for Coordinators field.
+    And I fill in "Name" with "Coordinators stakeholder" in the "Project coordinators" region
+    And I fill in "Acronym" with "Acronym of the Coordinator" in the "Project coordinators" region
+    And I fill in "Use existing media" with "Image 1" in the "Project coordinators" region
+    And I select "Belgium" from "Country" in the "Project coordinators" region
+    And I wait for AJAX to finish
+    And I fill in "Company" with "Ma société 1" in the "Project coordinators" region
```
Comment (yenyasinn) [inline tests/features/project.feature:35]: Can French language be used in the test content?

Comment (drishu) [reply tests/features/project.feature:35]: These are the types of values that will be used, so its nice to use them in the tests

Comment (ademarco) [reply tests/features/project.feature:37]: I'd use english on an english translation and french on a french one, let's stick to english unless we are testing a translation.

**Pull request merged**